podvm/Dockerfile.podvm_builder.rhel

```diff
@@ -24,6 +24,8 @@ RUN dnf groupinstall -y 'Development Tools' && \
     rm -rf /usr/local/go && tar -C /usr/local -xzf go${GO_VERSION}.linux-amd64.tar.gz && \
     rm -f go${GO_VERSION}.linux-amd64.tar.gz
 
+# set a correspond qemu-system-* named link to qemu-kvm
+RUN ln -s /usr/libexec/qemu-kvm /usr/bin/qemu-system-$(uname -m)
```

Comment:
IIUC, this will only work with qemu-system-x86_64, for other arches, you have to specify a qemu_binary path as well in your packer script, right ? I agree with @jtumber-ibm, I think the previous version was cleaner :)